repositories.yml

```diff
@@ -3,49 +3,173 @@ section:
   name: Conjur OSS Suite Release
   description: These are the primary repositories for Conjur Open Source and its SDK.
   categories:
-  - name: Conjur OSS Core
-    description: Conjur OSS Server and Deployment Tools
+  - name: Conjur Server
+    description: Conjur Core and Deployment Tools
     repos:
       - name: cyberark/conjur
         url: https://github.com/cyberark/conjur
         description: Conjur OSS server. Conjur comes built-in with custom authenticators
           for Kubernetes, OpenShift, AWS IAM, OIDC, and more.
         certification: "certified"
-        after: v1.3.5
+        version: v1.4.4
         upgrade_url: https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Deployment/Upgrade/upgrade-intro.htm
       - name: cyberark/conjur-oss-helm-chart
         url: https://github.com/cyberark/conjur-oss-helm-chart
         description: Helm chart for deploying Conjur OSS.
         version: v1.3.7
-      - name: cyberark/secretless-broker
-        url: https://github.com/cyberark/secretless-broker
-        description: Secretless Broker is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
-        version: v1.4.2
+        certification: "trusted"
 #      - name: cyberark/conjur-aws
 #        url: https://github.com/cyberark/conjur-aws
 #        description: AWS CloudFormation templates for deploying Conjur OSS.
+#        certification: "community"
+#        version: v{no tags, can't include in release}
```

Comment:
`version: v{no tags, can't include in release}` -> `version: nil # No tags, can't include in release`